cluster/images/kube-discovery/Makefile

```diff
@@ -0,0 +1,41 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build the hyperkube image.
+#
+# Usage:
+#   [ARCH=amd64] [REGISTRY="gcr.io/google_containers"] make (build|push) VERSION={some_released_version_of_kubernetes}
+
+REGISTRY?=gcr.io/google_containers
+ARCH?=amd64
+TEMP_DIR:=$(shell mktemp -d)
+
```

Comment:
Replace BASEIMAGE here based on ARCH like in the hyperkube image
amd64: busybox
arm: armel/busybox
arm64: aarch64/busybox
ppc64le/busybox